drivers/soundwire/intel.h

```diff
@@ -7,24 +7,39 @@
 /**
  * struct sdw_intel_link_res - Soundwire Intel link resource structure,
  * typically populated by the controller driver.
- * @pdev: platform_device
+ * @bus: bus handle
  * @mmio_base: mmio base of SoundWire registers
  * @registers: Link IO registers base
  * @shim: Audio shim pointer
  * @alh: ALH (Audio Link Hub) pointer
  * @irq: Interrupt line
  * @ops: Shim callback ops
  * @dev: device implementing hw_params and free callbacks
+ * @clock_stop_quirks: mask defining requested behavior on pm_suspend
```

Comment:
@plbossart Should clock_stop_quirks belong to this patch?

Reply:
The clock stop quirks are already defined on the SOF side so it doesn't really hurt to pass the information and use it later.